README.md

```diff
@@ -3,14 +3,29 @@
 ## Setup
 
 Install dependencies
-
 ```bash
 pip3 install -r requirements.txt
 ```
 
 ## Run
 
+To run you need proper `.env` file with secret tokens
+
+### Monolith version:
 ```bash
-source .env #env file with secret tokens
-uvicorn gtkm.main:app --reload
+uvicorn gtkm.main:app --reload --env-file .env
 ```
+
+### Microservice version:
+- Install nginx (https://www.nginx.com/resources/wiki/start/topics/tutorials/install/)
+- Windows:
+  - Place the unzipped nginx files in folder "nginx" in the main folder of the project
+  - Swap the file "nginx/conf/nginx.conf" for the one provided with the project
+  - Open cmd in the "nginx" folder
+  - to start nginx: `start nginx`
+- Linux:
+  - nginx files are located in /usr/bin/nginx
+  - to start nginx: `/usr/bin/nginx -c <path_to_conf_file>`
+- additional nginx commands:
+  - `nginx -s reload`
+  - `nginx -s quit`
```

Comment:
Czy sprawdzałeś tą instrukcję na linuxie?  
U mnie nie działa

```
nginx -c ./nginx/conf/nginx.conf 
nginx: [alert] could not open error log file: open() "/var/log/nginx/error.log" failed (13: Permission denied)
2022/05/19 18:37:15 [emerg] 76220#76220: open() "/usr/share/nginx/./nginx/conf/nginx.conf" failed (2: No such file or directory)
pawel@pop-os:/mnt/HDD/PROJEKTY/Projekty_przedmiotowe/git-to-know-me$ sudo nginx -c ./nginx/conf/nginx.conf 
nginx: [emerg] open() "/usr/share/nginx/./nginx/conf/nginx.conf" failed (2: No such file or directory)
pawel@pop-os:/mnt/HDD/PROJEKTY/Projekty_przedmiotowe/git-to-know-me$ sudo nginx -c /mnt/HDD/PROJEKTY/Projekty_przedmiotowe/git-to-know-me/nginx/conf/nginx.conf 
nginx: [emerg] open() "/mnt/HDD/PROJEKTY/Projekty_przedmiotowe/git-to-know-me/nginx/conf/mime.types" failed (2: No such file or directory) in /mnt/HDD/PROJEKTY/Projekty_przedmiotowe/git-to-know-me/nginx/conf/nginx.conf:9
```

I tak właściwie to po co to te `/usr/bin/nginx`? Jeśli nginx jest poprawnie zainstalowane to samo `nginx` powinno wystarczyć.